Notebooks/DC2 Run 1.1-test2 merged coadd color-color stellar locus.ipynb

```diff
@@ -0,0 +1,503 @@
+    "nbpresent": {
+     "id": "dacd2cd2-204b-43b9-9f68-447b52b581fd"
+    },
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "plot_color_color(galaxies, 'gmr', 'rmi')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_color_color(stars, 'gmr', 'rmi')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.scatter(df['r_mag'], 1/df['r_mag_err'], label='1/r_mag_err')\n",
+    "plt.xlabel('r')\n",
+    "plt.ylabel('1/r_err')\n",
+    "plt.axhline(bright_snr_threshold, linestyle='--', color='red', label='Bright star')\n",
+    "plt.axhline(snr_threshold, linestyle='--', color='orange', label='Good SNR')\n",
+    "plt.legend()\n",
+    "plt.xlim(15, 28)\n",
+    "# plt.ylim(0, 1.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_color_color(stars, 'rmi', 'imz')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "See, e.g., http://www.astroml.org/examples/datasets/plot_sdss_imaging.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "z = galaxies\n",
+    "plt.scatter(z['g_mag'] - z['r_mag'], z['g_mag'], marker='.')\n",
+    "plt.xlim(-1, +2)\n",
+    "plt.ylim(25, 15)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "z = stars\n",
+    "H, xedges, yedges = np.histogram2d(z['g_mag']-z['r_mag'], z['g_mag'],\n",
+    "                                    range=((-1, +2), (15, 25)), bins=31)\n",
+    "zi = H.T\n",
+    "xi = (xedges[1:] + xedges[:-1])/2\n",
+    "yi = (yedges[1:] + yedges[:-1])/2\n",
+    "\n",
+    "plt.pcolormesh(xi, yi, zi, cmap=cmap)\n",
+    "plt.contour(xi, yi, zi)\n",
+    "plt.xlabel('g-r')\n",
+    "plt.ylabel('g')\n",
+    "plt.ylim(25, 15)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbpresent": {
+     "id": "4791c937-9435-405d-8490-a4f9f6d80aa6"
+    }
+   },
+   "source": [
+    "## Appendix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generic Catalog Reader (GCR)\n",
+    "\n",
+    "After a little bit you'll find yourself wanting to ask interesting questions, like how well does the DC2 analysis recover the input sources?  Or easily loading in all of the patches of a tract.  A convenient framework for generalizing analyses of DESC-DC related catalogs is the Generic Catalog Reader:\n",
+    "\n",
+    "* GCR is a general utility\n",
+    "https://github.com/yymao/generic-catalog-reader\n",
+    "* The catalogs supporting DC2 are in:\n",
+    "https://github.com/LSSTDESC/gcr-catalogs\n",
+    "\n",
+    "See 'DC2_Static_Coadd_Catalog_reader.ipynb' notebook by Yao-Yuan Mao\n",
```

Comment:
Link to DC2_Static_Coadd_Catalog_reader.ipynb for easy navigation.

Reply:
The next line is that link.  Does it show up for you?

Reply:
Ah yes, for some reason I had thought that was something else. Not the notebook's fault, just a personal mental skip.

Reply:
👍